### ISSUE TITLE
[Bug]: Staedteservice.de / Ruesselsheim, Raunheim

### ISSUE THREAD

**Issue by franky1808** [mampfes/hacs_waste_collection_schedule#1446]
### I Have A Problem With:

A specific source, The integration in general

### What's Your Problem

The sensor has not delivered any values for a week now. I have adapted the integration according to the current instructions. 
Unfortunately without success. The example also does not work as it is written here.

### Source (if relevant)

staedteservice_de

### Logs

```Shell
"Diese Entität wird nicht mehr von der Integration waste_collection_schedule bereitgestellt. Wenn die Entität nicht mehr verwendet wird, lösche sie in den Einstellungen."
```

### Relevant Configuration

```YAML
As described in the example
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by 5ila5** [comment]
May be duplicate of #1400

There where some breaking changes on their side which where fixed in #1404 

> Fix will be part of release 1.44.0 or is available now as master (git) version (HACS: 3-dot menu -> re-download -> change version to master)

**Comment by franky1808** [comment]
> May be duplicate of #1400
> 
> There where some breaking changes on their side which where fixed in #1404
> 
> > Fix will be part of release 1.44.0 or is available now as master (git) version (HACS: 3-dot menu -> re-download -> change version to master. Don't forget to restart HA)

Totally true. Unfortunately, I completely overlooked it. Sorry! I'll close it. :-)
